reliability/single-fit/rum/src/test/kotlin/com/datadog/android/rum/integration/ManualTrackingRumTest.kt

```diff
@@ -395,6 +399,217 @@ class ManualTrackingRumTest {
 
     // endregion
 
+    // region AddViewLoading time
+
+    @OptIn(ExperimentalRumApi::class)
+    @RepeatedTest(16)
```

Comment:
Why do we want to repeat it 16 times given that there is only 2 execution branches for the given input - with overwrite and without. For the test above 16 (or repeat the test in general) makes sense, because there is more code paths there for the given input, I guess.

Reply:
I guess 1 time is faster and doesn't change anything, just want to check with @xgouchet what was his intention with the previous tests and in case we're good I will change it to one single test.

Reply:
`RepeatedTest` makes sense if there is a big variety in the input parameters which can lead to many different code paths taken. But here the only parameter which _can_ lead (I'm not sure, didn't dig into the code) to the different code paths is the `overwrite` flag, which is simply binary.

Reply:
yes...it make sense, let me change that to a single test.